cibyl/utils/dicts.py

```diff
@@ -33,8 +33,6 @@ def subset(dictionary, keys):
     for key in keys:
         # Do not crash if a key is not present
         if key not in dictionary:
-            message = "Ignoring key '%s' not found in dictionary: %s"
-            LOG.debug(message, key, dictionary)
```

Comment:
Why so? It is at 'debug' level and notifies of an easy to miss mistake: an incorrect key name.

Reply:
For me, it's logged more than 30 times per run. It would be nice if we could make it shorter and concise somehow